models/engine/db_storage.py

```diff
@@ -74,3 +74,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve one object"""
+        if cls.__name__ in classes and id is not None:
+            return self.__session.query(classes[cls.__name__])\
+            .filter(classes[cls.__name__].id == id).first()
+        else:
+            return None
```

Comment:
Protip: Same sug that in [file_storage comment](https://github.com/02KellyV/AirBnB_clone_v3/pull/1/files#r416317606).